### ISSUE TITLE
Move docs files into their own folder

### ISSUE THREAD

**Issue by kylejrp** [kognise/water.css#88]
@jonaskuske had a great suggestion in #71 to move all the docs files to their own folder. I think this will help clean up the project directory.

However, this change will break the Netlify configuration, as it's looking for `index.html` and not `docs\index.html`. @kognise will have to update the Netlify configuration as we make this change.

**Comment by kimulaco** [comment]
@kognise @kylejrp @jonaskuske 
I saw some issues and thought that I needed to move files other than docs files.(#89 #84 etc...)
We will also need to change the src directory to make a JavaScript build environment.
And that's related to moving docs files.

I think the following directory is common. (Those that have not been changed are omitted.)


**now**

```
├ dist/*.css
├ src/
│  ├ builds/*.css
│  ├ parts/*.css
│  └ *.css
├ index.html
├ scripts.js
└ ...
```

**after**


```
├ dist/*.css
├ docs/
│ ├ index.html
│ ├ css/*.css  <= Needed here too?
│ └ js/*.js
├ src/
│ ├ css/
│ │  ├ builds/*.css
│ │  ├ parts/*.css
│ │  └ *.css
│ └ js/*.js
└ ...
```

Please let us know if you have any ideas for your opinion or response policy!

**Comment by jonaskuske** [comment]
@kimulaco 
What's `src/js/*` in this case?
If it's the JavaScript for the docs I must say I'm against this, I feel like the `src` directory should only contain the source files for `water.css` itself, not accompanying documentation fluff.

But you're right that we need separate places for compiled docs and source docs if we want a build chain there. Alternatives would be:

  - Use `docs/*` as source directory and compile to either `dist/docs/` or `docs/dist/`. This way it'd be clear that the docs in the respective `dist` folder are "just" compiled distribution files and one mustn't edit them. Downside is that some tools like GitHub pages expect `docs/` to contain the actual documentation, not uncompiled source files – but I don't see us leaving Netlify and migrating back to GitHub Pages anytime soon, so I don't think that's too important.
  - Use `src/docs/*` as source dir, compile to `docs/`. This way we got our documentation files within `src/` which I'm against in general, but since the files are in a folder called `docs/` instead of src directories with generic names like `js/`, I think I'd be fine with that, as this way they can't be confused as part of the actual `water.css` source. Problem with this is that now the intuitive and instantly-visible `docs/` folder is NOT to the place for our contributors to edit the documentation.

**TL;DR: I'm in favor of keeping a clear separation between the source of `water.css` itself and its documentation, the best way to achieve this can be discussed.**

**Comment by kimulaco** [comment]
I considered these as one project and had the idea of saving the pre-build JavaScript resources used for the document in `src/js/*`.
However, as @jonaskuske says, I agree to separate the resources of the library itself and the documentation!!

**Comment by kognise** [comment]
This was fixed in #90 which has been merged. <3
